tests/helpers/docker/ganache.py

```diff
@@ -106,3 +106,21 @@ def wait(self, max_attempts: int = 15, sleep_rate: float = 1.0) -> bool:
                 )
                 time.sleep(sleep_rate)
         return False
+
+
+class GanacheForkDockerImage(GanacheDockerImage):
+    """Extends GanacheDockerImage to enable forking"""
+
+    NETWORK: str = "ropsten"
+    BLOCK_NUMBER: int = 11290000
+
+    def _build_command(self) -> List[str]:
+        """Build command."""
+
+        cmd = [
+            "--wallet.deterministic=true",
+            f"--fork.network={self.NETWORK}",
+            f"--fork.blockNumber={self.BLOCK_NUMBER}",
+        ]
```

Comment:
wrt parent class, we do not set gas limit and account/balances:

https://github.com/valory-xyz/open-autonomy/blob/35a90b02d2b30371994796a0935bb2568ef4f83e/tests/helpers/docker/ganache.py#L73-L81

Probably the solution is to call `super()._build_command()` and then append the new options?

Reply:
@0xArdi agree with @marcofavorito here, any reason you see why we shouldn't do that?

Reply:
`--wallet.accounts` takes precedence over `--wallet.deterministic=true`, but that's not a problem.